### PR TITLE
Henrynunez112/svg

### LINKED ISSUE #103: Remove react-native-svg and svgs libraries
This library adds 86k (gzipped) to the bundle and we could be just writing plain svg in react. 

remove it and if possible the "shared/svg" file it is in

### PR DESCRIPTION
**Pull request by Henrynunez112**
**Description**

What is the current behavior, and what is the updated/expected behavior with this PR?

Was working on the converting the react-native-svg to regular svg to save some space. It's the initial steps I personally don't know if I need to do more work. Feedback is welcome, encouraged actually.

Fixes #103

### CODE REVIEW

File: src/about/Tetris.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import * as React from 'react'
 import { colors } from 'src/styles'
-import Svg, { Path } from 'svgs'
+import { Path } from 'svgs'
```
Comment (aaronmgdr) [inline src/about/Tetris.tsx:3]: we want to get rid of these too.

File: src/alliance/AllianceLogo.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import * as React from 'react'
-import Svg, { Path } from 'svgs'
+import { Path } from 'svgs'
 export default React.memo(function AllianceLogo() {
   return (
-    <Svg width="47" height="48" viewBox="0 0 47 48" fill="none">
+    <svg width="47" height="48" viewBox="0 0 47 48" fill="none">
       <Path
```
Comment (aaronmgdr) [inline src/alliance/AllianceLogo.tsx:6]: all svg elements should be converted to lowercase "dom" elements. 

so. `<Path> =>  <path>`

File: src/icons/Android.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { colors } from 'src/styles'
-import Svg, { ClipPath, Defs, G, Path, Rect } from 'svgs'
+import  { ClipPath, Defs, G, Path, Rect } from 'svgs'
```
Comment (aaronmgdr) [inline src/icons/Android.tsx:3]: remove imports and convert all components to elements

File: src/alliance/AllianceLogo.tsx (hunk 2 above)
Comment (Henrynunez112) [reply src/alliance/AllianceLogo.tsx:6]: Cool cool thank you for this.

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**